src/MSALWrapper/AuthFlow/AuthFlowExecutor.cs

```diff
@@ -48,9 +53,11 @@ public async Task<AuthFlowResult> GetTokenAsync()
                 this.logger.LogDebug($"Starting {authFlowName}...");
 
                 var attempt = await authFlow.GetTokenAsync();
+                this.SendTelemetryEvent(attempt, authFlowName);
+
                 if (attempt == null)
                 {
-                    var oopsMessage = $"Auth flow '{authFlow.GetType().Name}' returned a null AuthFlowResult.";
+                    var oopsMessage = $"Auth flow '{authFlowName}' returned a null AuthFlowResult.";

```

Comment:
How is the `authFlowName` set?

Reply:
This change is not highlighted because you added the same change I think and I merged your changes into this PR.